src/cmd/vm.rs

```diff
@@ -24,5 +36,70 @@ pub fn run(args: &Args) -> Result<()> {
         ));
     }
 
+    match &args.nested {
+        SubCommand::Connect(args) => run_connect(args),
+        SubCommand::Setup(args) => run_setup(args),
+        SubCommand::Start(args) => run_start(args),
+        SubCommand::Push(args) => run_push(args),
+    }
+}
+
+#[derive(FromArgs, PartialEq, Debug)]
+/// connects to a running betty instance via SSH
+#[argh(subcommand, name = "connect")]
+pub struct ArgsConnect {
+    /// extra arguments
+    #[argh(option)]
+    extra_args: Vec<String>,
+}
+
+fn run_connect(args: &ArgsConnect) -> Result<()> {
+    let _args = args;
+
+    Ok(())
+}
+
+#[derive(FromArgs, PartialEq, Debug)]
+/// run first time setup, installs necessary dependencies
+#[argh(subcommand, name = "setup")]
+pub struct ArgsSetup {
+    /// extra arguments
+    #[argh(option)]
+    extra_args: Vec<String>,
+}
+
+fn run_setup(args: &ArgsSetup) -> Result<()> {
```

Comment:
just rename to _args, no need to set it on a let

Reply:
Got it, I fixed arguments!